docs/meetups/wtd-community-organizing.rst

```diff
@@ -0,0 +1,149 @@
+Volunteer to become a Write the Docs Community Organizer!
+=========================================================
+
+We are connecting the global Write the Docs (WTD) community by hosting local meetup and similar community events near you.
+
+Who are Write the Docs community organizers?
+--------------------------------------------
+
+Write the Docs communinity organizers are volunteers who build and support WTD communities around the world. Their goal is to create a space for documentarians to build relationships and support each other in their local, or time zone based, communities.
+
+Interested in volunteering as an organizer? Read more and `apply today <https://forms.gle/Jx9CniuwEcLrEYPJ7>`__!
+
+Write the Docs Meetups and similar communities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Write the Docs brings together local documentarians and tech enthusiasts through our network of more than 20 meetup groups as well as similar community sites.
+
+Our meetup groups and community events are welcoming spaces for you to learn, connect, and grow in your journey as a documentarian.
+
+
+The path to being an organizer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All of our organizers are volunteers local to the area, or time zone they serve.
+
+1. Determine if you are interested and up to the task
```

Comment:
```suggestion
1. Determine if you are interested and have the capacity for this role
```

Reply:
Do we want to describe roughly how much of a time commitment it takes? Maybe link to the meetups page where we discuss the minimum frequency of meetups required to maintain WTD support